src/desktop/secret.rs

```diff
@@ -19,8 +19,12 @@
 //! }
 //! ```
 
+#[cfg(not(feature = "async-std"))]
```

Comment:
tokio doesn't have an equivalent for this?

Reply:
it does, yes.